# Service to Set the motor position
int64 enc      # Position of the motor encoder
---
int64 ret      # Set to 1 (success) or 0 (error)
string msg     # Error description